services/blockchain-indexer/shared/utils/array.js

```diff
@@ -44,8 +44,17 @@ const sortComparator = (sortParam) => {
 	return comparator;
 };
 
+const isPatternInCollection = (collection, pattern) => {
```

Comment:
pattern mostly refers to regex checking. Since we are doing substring checking here maybe we could name it something more relevant. ex: isSubStringOf, isSubStringOfArray